samza-kv/src/main/scala/org/apache/samza/storage/kv/BaseKeyValueStorageEngineFactory.scala

```diff
@@ -32,15 +32,15 @@ import org.apache.samza.storage.{StorageEngine, StorageEngineFactory, StorePrope
 import org.apache.samza.system.SystemStreamPartition
 import org.apache.samza.task.MessageCollector
 import org.apache.samza.util.ScalaJavaUtil.JavaOptionals
-import org.apache.samza.util.{HighResolutionClock, ScalaJavaUtil}
+import org.apache.samza.util.{HighResolutionClock, Logging}
 
 /**
   * A key value storage engine factory implementation
   *
   * This trait encapsulates all the steps needed to create a key value storage engine. It is meant to be extended
   * by the specific key value store factory implementations which will in turn override the getKVStore method.
   */
-trait BaseKeyValueStorageEngineFactory[K, V] extends StorageEngineFactory[K, V] {
+trait BaseKeyValueStorageEngineFactory[K, V] extends StorageEngineFactory[K, V] with Logging {
```

Comment:
Minor: Is the `Logging` trait still needed?

Reply:
No. Removed.